esx_service/utils/auth.py

```diff
@@ -28,6 +28,7 @@
 CMD_REMOVE = 'remove'
 CMD_ATTACH = 'attach'
 CMD_DETACH = 'detach'
+CMD_LIST = 'list'
```

Comment:
I had removed this in my last PR. Any reason we are re-introducing? 
See executeRequest. Auth checks are bypassed for 'list' command.